File: FavoDeMel-API/src/FavoDeMel.Domain/Entities/Pedido.cs

using FavoDeMel.Domain.Enums;
using FavoDeMel.Infrastructure.Data;
using System;

namespace FavoDeMel.Domain.Entities
{
    public class Pedido : Entity
    {
        public string Nome { get; set; }
        public Guid ComandaId { get; set; }
        public int Quantidade { get; set; }
        public string Observacao { get; set; }
        public EnumSituacaoPedido Situacao { get; set; }
    }
}

File: FavoDeMel-UI/src/app/shared/models/pedido.ts
import { SituacaoPedido } from '../enums/situacao-pedido.enum';

export interface Pedido {
  id: string;
  nome: string;
  comandaId: string;
  observacao: string;
  quantidade: number;
  situacao: SituacaoPedido;
  dataCriacao: Date;
  dataUltimaAlteracao: Date;
}